examples/todo/index.js

```diff
@@ -12,6 +12,7 @@ if (require.main === module) {
   const config = {
     rest: {
       port: +process.env.PORT || 3000,
+      // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing
```

Comment:
The ts eslint rules like `@typescript-eslint/prefer-nullish-coalescing` shouldn't check js file

Reply:
Is it required for the build to pass? We already have this override - https://github.com/strongloop/loopback-next/blob/master/packages/eslint-config/eslintrc.js#L158

Reply:
@raymondfeng Not really, our code base is good, and adding this line results in lint error...
But the todo example created by `lb4 example` complains:

```sh
lb4 example (choose todo)
cd loopback4-example-todo
npm i
npm run lint

> @loopback/example-todo@1.9.4 lint /Users/jannyhou/Desktop/2019/ui/test/loopback4-example-todo
> npm run prettier:check && npm run eslint
> @loopback/example-todo@1.9.4 prettier:check /Users/jannyhou/Desktop/2019/ui/test/loopback4-example-todo
> npm run prettier:cli -- -l
> @loopback/example-todo@1.9.4 prettier:cli /Users/jannyhou/Desktop/2019/ui/test/loopback4-example-todo
> lb-prettier "**/*.ts" "-l"
> @loopback/example-todo@1.9.4 eslint /Users/jannyhou/Desktop/2019/ui/test/loopback4-example-todo
> lb-eslint --report-unused-disable-directives .
/Users/jannyhou/Desktop/2019/ui/test/loopback4-example-todo/index.js
  15:30  error  Prefer using nullish coalescing operator (`??`) instead of a logical or (`||`), as it is a safer operator  @typescript-eslint/prefer-nullish-coalescing

✖ 1 problem (1 error, 0 warnings)
  1 error and 0 warnings potentially fixable with the `--fix` option.
```

Reply:
I see. I wonder why https://github.com/strongloop/loopback-next/blob/master/packages/eslint-config/eslintrc.js#L158 does not exclude js files. Can you check which version of `@loopback/eslint-config` is pulled in?

Reply:
@raymondfeng It's already on the latest released version `@loopback/eslint-config@5.0.1`

Reply:
Maybe add a file similar as https://github.com/strongloop/loopback4-example-shopping/blob/d0f6d7543c1a861dc4a4281b6ef7b8ac8d1085dc/.eslintrc.js?

Reply:
@raymondfeng 👍 tested the config file, I think the necessary config is `extends: ['@loopback/eslint-config']`

Details verified in comment https://github.com/strongloop/loopback-next/pull/4431#issuecomment-574380166